.github/fabricbot.json

```diff
@@ -1463,7 +1463,7 @@
           ],
           "mentionees": [
             "ctstone",
-            "anrothMSFT"
+            "vkurpad"
```

Comment:
out of curiosity, si there a way to keep both files synced so we don't forget to update this in the future?

Reply:
Not currently, this is one of the big weaknesses of Fabric Bot.   We're looking at some alternatives so that we can move these out of CODEOWNERS and into a single data file, but that's a bit of a longer-term project.